vibe_code_apps_20/wattlab/studio/eui_compare.py

```
@@ -0,0 +1,159 @@
+        }
+    )
+    if model_eui_kbtu_ft2 is not None:
+        model_cmp = compare_eui(float(model_eui_kbtu_ft2), ptype)
+        scale = float(prototype_area_scale) if prototype_area_scale else None
+        note = (
+            f"{model_label} on prototype footprint (~{PROTOTYPE_AREA_FT2_NOMINAL:,.0f} ft²)"
+        )
+        if target_floor_area_ft2:
+            note += f"; target site {float(target_floor_area_ft2):,.0f} ft²"
+        if scale and scale > 1.05:
+            note += f"; scale≈{scale:.2f}× — do not treat raw kWh as site total"
```

Ruff (0.15.21)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)